gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -235,9 +235,6 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-form-element-border">
               The standard color for the borders of form fields (such as text inputs) in their default state.
             </ColorDocRow>
-            <ColorDocRow colorVar="--nx-color-form-element-border-pristine">
-              The standard color for the borders of form fields (such as text inputs) in their pristine state.
```

Comment:
Removing the actual variable would be a breaking change.  Instead, mark it as deprecated and explain that we no longer use it.

Reply:
> > Updated to deprecated: [867bfa9](https://github.com/sonatype/sonatype-react-shared-components/commit/867bfa96276d741db73a1dc17d89e8ba58d04042)
> 
> > **and explain that we no longer use it.**

Sorry, forgot to add the explanation: Let me know if this is sufficient: e44e2525b03f6b0729c7fcc58a68e78834958efa

Reply:
> Sorry, forgot to add the explanation: Let me know if this is sufficient: e44e252

I feel like it's a little bit confusing the way you currently have it.  The simplest fix I think would be to separate the description of what the variable is for out of the explanation of why it's deprecated.  i.e. move the second sentence out of the alert.